isis/src/base/objs/StretchBlob/StretchBlob.cpp

```diff
@@ -51,67 +60,52 @@ namespace Isis {
     return m_stretch;
   }
 
-  /**
-   * Read saved Stretch data from a Cube into this object. 
-   *  
-   * This is called by Blob::Read() and is the actual data reading function 
-   * ultimately called when running something like cube->read(stretch);  
-   * 
-   * @param is input stream containing the saved Stretch information
-   */
-  void StretchBlob::ReadData(std::istream &is) {
-    // Set the Stretch Type
-     m_stretch.setType(p_blobPvl["StretchType"][0]);
```

Comment:
Where does this happen in updated code?

Reply:
I was thinking that this happened during the parsing step, i.e.:

```m_stretch.Parse(QString::fromStdString(stringFromBuffer));```